src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,35 @@
+type ErrorType = 'load' | 'add' | 'delete' | 'update' | '';
```

Comment:
you alreade has same type in App component.
you can make some global file types.ts and keep there all types